server/server/views/user.py

```diff
@@ -32,8 +32,13 @@ def get(self):
     def put(self):
         request = self.request
         data = request.validated
-        
+        update_status = False
+
+        response = {'is_updated': False}
```

Comment:
I'd rather leave `success` as it was agreed to use this field to indicate that something went wrong.